src/ztex/fpga-md5crypt/README.md

```diff
@@ -0,0 +1,173 @@
+## Overview
+
+- md5crypt ($1$) for ZTEX 1.15y board allows candidate passwords up to
+64 bytes, equipped with on-board candidate generator and comparator.
+It's also able to compute phpass-MD5 hashes.
```

Comment:
It'd be good to clarify whether the same limit of 64 characters applies to phpass as well.

Reply:
Candidates of lengths up to and including 64 bytes can be passed to phpass program, there's enough space in the memory, handled w/o problems. However I decided to keep phpass(-cpu) limit on the length because I don't have example hashes with plaintexts up to & including 64 bytes long. I can set PLAINTEXT_LENGTH equal to 64 for phpass, that would perform but I can't verify correctness. 
Could you provide example hashes (from plaintexts <= 64 bytes)? (could not find any on the Internet). Alternatively I can set-up the software/environment where such hashes are used, produce hashes, unfortunately I'm running out of the timeline previously agreed for (md5crypt|phpass)-ztex development.

Reply:
Oh, looking at the code I see that the limit of 39 is getting inherited from `phpass_common.h`. This isn't a terribly low limit, but 64 would be nicer - and we need to document the actual limit either way. Yes, I'll generate and provide a test hash for a 64 chars string. Thanks!

Reply:
`test012345678901234567890123456789012345678901234567890123456789` `$P$9IQRaTwmfexDe7M./J5j8/buI4F3sX0`